### PR TITLE
feat: update codeowners

### PR DESCRIPTION
**Pull request by estrattonbailey**
Bop it 💥

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]
i'm free 💹

File: CODEOWNERS [hunk 1]
```diff
@@ -1,80 +1,65 @@
...
-metro.transform.js @brunobar79 @osdnk @estebanmino
-package.json @brunobar79 @estebanmino @nickbytes
+package.json @salman-rb @skylarbarrera @estrattonbailey
+.eslintignore @salman-rb @estrattonbailey @skylarbarrera
+.eslintrc.js @salman-rb @estrattonbailey @skylarbarrera
+.gitattributes @brunobar79 @salman-rb @estrattonbailey @skylarbarrera
+.gitignore @salman-rb @estrattonbailey @skylarbarrera
+.nvmrc @salman-rb @estrattonbailey @skylarbarrera
+.prettierrc.js @salman-rb @estrattonbailey @skylarbarrera
+.watchmanconfig @jkadamczyk
```
Comment (brunobar79) [inline CODEOWNERS:47]: I think we can drop this one

File: CODEOWNERS [hunk 2]
```diff
@@ -1,80 +1,65 @@
...
+.yarnrc @estrattonbailey @skylarbarrera
+audit-ci.json @brunobar79 @salman-rb @skylarbarrera @estebanmino @estrattonbailey
+babel.config.js @brunobar79 @estrattonbailey @jkadamczyk @skylarbarrera
```
Comment (brunobar79) [inline CODEOWNERS:50]:
```suggestion
babel.config.js @estrattonbailey @jkadamczyk @skylarbarrera
```

File: CODEOWNERS [hunk 3]
```diff
@@ -1,80 +1,65 @@
...
+globals.d.ts @estrattonbailey @skylarbarrera
+yarn.lock @salman-rb @estrattonbailey @skylarbarrera
+tsconfig.json @salman-rb @estrattonbailey @skylarbarrera
+hardhat.config.js @salman-rb @skylarbarrera @estebanmino
```
Comment (brunobar79) [inline CODEOWNERS:54]:
```suggestion
hardhat.config.js @salman-rb @skylarbarrera
```

Removing Esteban, you can ping us if needed

File: CODEOWNERS [hunk 4]
```diff
@@ -1,80 +1,65 @@
...
-tsconfig.json @brunobar79 @osdnk @estebanmino @nickbytes
+globalVariables.js @estrattonbailey @skylarbarrera
+shim.js @brunobar79 @salman-rb @estrattonbailey @skylarbarrera @jkadamczyk
+index.js @salman-rb @estrattonbailey @jkadamczyk @skylarbarrera
+src/App.js @salman-rb @estrattonbailey @skylarbarrera
 
 # SECURITY / APP INFRA
-yarn.lock @brunobar79 @estebanmino @nickbytes
+metro.config.js @skylarbarrera @estrattonbailey @jkadamczyk
+metro.transform.js @skylarbarrera @estrattonbailey @jkadamczyk
```
Comment (brunobar79) [inline CODEOWNERS:64]:
```suggestion
metro.transform.js @skylarbarrera @estrattonbailey @jkadamczyk @brunobar79 
```

File: CODEOWNERS [hunk 5]
```diff
@@ -1,80 +1,65 @@
...
-src/design-system @markdalgleish @jxom
+src/design-system @jxom @estrattonbailey
 
 # Handlers
 src/handlers/aesEncryption.ts @brunobar79 @estebanmino
 src/handlers/authentication.ts @brunobar79 @estebanmino
-src/handlers/cloudBackup.ts @brunobar79 @skylarbarrera @estebanmino
-src/handlers/fedora.ts @osdnk @estebanmino @terrysahaidak
+src/handlers/cloudBackup.ts @brunobar79 @skylarbarrera @estebanmino @jkadamczyk
+src/handlers/fedora.ts @estebanmino
```
Comment (brunobar79) [inline CODEOWNERS:24]:
```suggestion
src/handlers/fedora.ts @estebanmino @brunobar79 
```

**Review by brunobar79 — APPROVED** [review]
🫡

**Review by nickbytes — APPROVED** [review]

**Pull request merged**